### ISSUE TITLE
Create new buildings energy use chart that shows total energy use

### ISSUE THREAD

**Issue by minshudeng** [Energy-Innovation/eps-us#173]
In the web app, there are currently three ways to visualize building energy use: by building type, by building component, and by energy source. For scoping out building initiatives, Jenny and I were wondering if it'd be possible to create a fourth chart that shows total energy use. That way we can quickly see the total instead of having to download the data and sum it manually.

(Jenny and I discussed this with Robbie who mentioned it might be a quick fix/change, but no worries if it's too involved.)

**Comment by robbieorvis** [comment]
Just to elaborate, we talked about creating total final energy use charts in addition to total primary energy use charts, so a total final energy consumption chart by sector would solve this I think.

On Jul 1, 2021, at 6:58 PM, minshudeng ***@***.***> wrote:

﻿

In the web app, there are currently three ways to visualize building energy use: by building type, by building component, and by energy source. For scoping out building initiatives, Jenny and I were wondering if it'd be possible to create a fourth chart that shows total energy use. That way we can quickly see the total instead of having to download the data and sum it manually.

(Jenny and I discussed this with Robbie who mentioned it might be a quick fix/change, but no worries if it's too involved.)
